src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommand.cs

```diff
@@ -178,5 +202,30 @@ public override int Execute()
                 return 1;
             }
         }
+
+        private int Validate()
+        {
+            if (_binPath != null && _global)
+            {
+                throw new GracefulException("Cannot have global and bin-path as opinion at the same time.");
+            }
+
+            if (_configFilePath != null && !File.Exists(_configFilePath))
+            {
+                throw new GracefulException(
+                    string.Format(
+                        LocalizableStrings.NuGetConfigurationFileDoesNotExist,
+                        Path.GetFullPath(_configFilePath)));
+            }
+
+            // Prevent installation if any version of the package is installed
+            if (_toolPackageStore.GetInstalledPackages(_packageId).FirstOrDefault() != null)
```

Comment:
So we can't do `dotnet install foo` if a global tool called `foo` is already installed, correct?

Reply:
good catch